diff --git a/pathnames.h b/pathnames.h
index 04b19b6..8f9c456 100644
--- a/pathnames.h
+++ b/pathnames.h
@@ -47,6 +47,7 @@
 			 * LOG_FILE or SYSLOG is defined, we don't log.  If
 			 * both are defined, we log both ways.
 			 */
+#ifndef DISABLE_ALLOW_DENY_FILES
 #ifdef DEBIAN
 #define	ALLOW_FILE	"/etc/cron.allow"		/*-*/
 #define DENY_FILE	"/etc/cron.deny"		/*-*/
@@ -54,6 +55,7 @@
 #define	ALLOW_FILE	"allow"		/*-*/
 #define DENY_FILE	"deny"		/*-*/
 #endif
+#endif
 /* #define LOG_FILE	"log"		  -*/
 
 			/* where should the daemon stick its PID?
